cypress/integration/event-submission-form.js

```diff
@@ -31,31 +31,31 @@ describe('Event Submission Form DOM',() => {
           .should('have.length', 9)
 
         cy.get('#category option').eq(0)
-          .should('have.value', 'animal-care')
+          .should('have.value', 'Animal Care')
```

Comment:
Semi expecting tests around this section to fail. I think I've been confused about how we've been handling attributes on this data list options. Tests may not fail and I may just be confused. Just saying it outloud right now in case I'm either right or wrong!